deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -145,72 +145,70 @@ services:
     tmpfs:
       - /tmp:rw,noexec,nosuid
     environment:
+      - NATS_HOST=nats
       - OADB_USER
       - OADB_PW
       - OADB_SERVICE=mock
-      - JAVA_OPTS=-Xms128m -Xmx256m -XX:+UseG1GC -Dlogging.level.mushop.orders=TRACE -Djavax.net.ssl.trustStorePassword=Welcome123 -Djavax.net.ssl.keyStorePassword=Welcome123 -Djava.security.egd=file:/dev/urandom -Dspring.zipkin.enabled=false
+      - JAVA_OPTS=-Xms32m -Xmx256m -XX:+UseG1GC  -Djavax.net.ssl.trustStorePassword=Welcome123 -Djavax.net.ssl.keyStorePassword=Welcome123 -Djava.security.egd=file:/dev/urandom -Dspring.zipkin.enabled=false
 
-  # Shipping service
-  shipping:
-    image: iad.ocir.io/oracle/ateam/mushop-shipping:1.0.0
-    hostname: shipping
+  payment:
+    image: iad.ocir.io/oracle/ateam/mushop-payment:0.0.1
+    hostname: payment
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
-    tmpfs:
-      - /tmp:rw,noexec,nosuid
-    environment:
-      - reschedule:on-node-failure
-      - OCI_TENANT_ID
-      - OCI_COMPARTMENT_ID
-      - OCI_REGION
-      - OCI_USER_ID
-      - OCI_FINGERPRINT
-      - OCI_API_KEY
 
-  # OCI Stream
-  stream:
-    image: iad.ocir.io/oracle/ateam/mushop-stream:1.0.0
-    hostname: stream
+  user:
+    image: iad.ocir.io/oracle/ateam/mushop-user:1.1.0
+    hostname: user
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
     environment:
-      - reschedule:on-node-failure
-      - OCI_TENANT_ID
-      - OCI_COMPARTMENT_ID
-      - OCI_REGION
-      - OCI_USER_ID
-      - OCI_FINGERPRINT
-      - OCI_API_KEY
+      - PORT=80
+      - OADB_USER=mock
+      - OADB_PW=mock
+      - OADB_SERVICE=mock
 
-  payment:
-    image: iad.ocir.io/oracle/ateam/mushop-payment:0.0.1
-    hostname: payment
+  nats:
+    image: nats:2.1.2
+    ports:
+      - 4222:4222
+      - 6222:6222
+      - 8222:8222
+    hostname: nats
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
+    environment:
+      - PORT=80
+      - OADB_USER=mock
+      - OADB_PW=mock
+      - OADB_SERVICE=mock
 
-  user:
-    image: iad.ocir.io/oracle/ateam/mushop-user:1.1.0
-    hostname: user
+# Fulfillment service
+  fulfillment:
+    image: iad.ocir.io/oracle/ateam/mushop-fulfillment:1.0.0
+    hostname: fulfillment
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
+    tmpfs:
+      - /tmp:rw,noexec,nosuid
     environment:
-      - PORT=80
-      - OADB_USER=mock
-      - OADB_PW=mock
-      - OADB_SERVICE=mock
+      - reschedule:on-node-failure
+      - NATS_HOST=nats
```

Comment:
Do we also need to specify `nats` connection ports?

Reply:
Not really required, but we can. Nats docker container is multiport, and exposes 3 standard ports for nats. 4222,6222,8222. we can obviously give users the ability to change that, but I though there wasn't really a reason.